goa/Classes/ext/LipiTKNode.cpp

```diff
@@ -90,7 +90,10 @@ bool LipiTKNode::initialize(int width, int height, Point position) {
     
         #elif (CC_TARGET_PLATFORM == CC_PLATFORM_ANDROID)
             //initialize lipiTK
-             path = "/storage/emulated/0/Android/data/com.maq.xprize.chimple.hindi/files/res/android-lipitk";
+                std::string fileName = "res/android-lipitk/projects/lipiengine.cfg";
+                std::string path1 = FileUtils::getInstance()->fullPathForFilename(fileName);
```

Comment:
Give a good name and never use path1 or path2 or path 3 etc.

Reply:
I have removed the variable path1